server/graphql/v2/mutation/TierMutations.js

```diff
@@ -93,6 +93,9 @@ const tierMutations = {
       tier: {
         type: new GraphQLNonNull(TierReferenceInput),
       },
+      stopRecurringContributions: {
+        type: GraphQLBoolean,
+      },
```

Comment:
:ice_cream: Personal preference, to remove any ambiguity

```suggestion
      stopRecurringContributions: {
        type: new GraphQLNonNull(GraphQLBoolean),
        defaultValue: false,
      },
```

_@hdiniz Note on this feedback:_ Whenever I use :ice_cream: / "Personal preference", you don't have to iterate on the feedback; do so only if you agree and feel like it's a good fit for the PR.